server/routes/index.js

```diff
@@ -23,6 +23,8 @@ routes.get('/users/requests/:requestId', UserRequestController.getRequestByReque
 // Modify Request details by requestId
 routes.put('/users/requests/:requestId', Validate.checkRequestInputs, UserRequestController.editRequest);
 // Sign user up
-routes.post('/auth/signup', Validate.signUpValidate, CheckExistingUser.checkExistingUser, UsersController.createUsers);
+routes.post('/auth/signup', Validate.signUpValidate, CheckExistingUser.checkExistingUser, UsersController.userSignUp);
```

Comment:
Explain this **CheckExistingUser.checkExistingUser**  middleware

Reply:
This middleware checks if the user exist in the database